src/applications/mhv/medical-records/tests/containers/App.unit.spec.jsx

```diff
@@ -66,7 +67,20 @@ describe('App', () => {
       dismissedDowntimeWarnings: [],
     },
   };
-
+  const downtime = maintenanceWindows => {
```

Comment:
probably not a good time with the move to the platform mhv dir, but maybe we could get out the DRYer and move this to a helper function later on :-)

Reply:
I was thinking that we could even abstract the entire test for Maintenance Windows into a helper function that takes an array of service IDs and a boolean for if the Maintenance Window should be shown or not. If we move to custom Maintenance Banners we would have to rethink that, maybe pass in an array of assertions or something; future! It was pretty gut wrenching copy pasta-ing those very similar tests so many times!